server/render.js

```diff
@@ -33,7 +33,7 @@ export async function render (url, ctx = {}, {
     return (staticMarkup ? renderToStaticMarkup : renderToString)(app)
   })
 
-  const head = Head.rewind() || []
+  const head = Head.rewind() || defaultHead()
```

Comment:
We still want to use the default head tags even if the page does not have a `<Head>`.